.travis.yml

```diff
@@ -16,3 +16,25 @@ script:
 cache:
   directories:
     - "node_modules"
+
+deploy:
```

Comment:
Please put the deploy section above the cache section, like we have in Android
https://github.com/NASAWorldWind/WorldWindAndroid/blob/develop/.travis.yml